show/plugins/barefoot.py

```diff
@@ -4,6 +4,7 @@
 import json
 import subprocess
 from sonic_py_common import device_info
+from sonic_py_common.general import getstatusoutput_noshell_pipe, check_output_pipe
 
 @click.group()
 def barefoot():
```

Comment:
>barefoot [](http://example.com/codeflow?start=4&length=8)

This file need to reviewed by barefoot. Suggest further split into a smaller PR.

Reply:
will do